agent_pool.go

```diff
@@ -70,6 +70,8 @@ type AgentPoolListOptions struct {
 	// Optional: A list of relations to include. See available resources
 	// https://www.terraform.io/cloud-docs/api-docs/agents#available-related-resources
 	Include []AgentPoolIncludeOpt `url:"include,omitempty"`
+
+	Query string `url:"q,omitempty"`
```

Comment:
Missing the code documentation for this field 💅

Reply:
@sebasslash added, thanks! I also updated the changelog...I notice that 1.2.0 was still marked as unreleased, so I updated that as well, please let me know if that is not correct!